### PR TITLE
Pre-release r1.1 of v0.1.0-rc.1

### LINKED ISSUE #17: Create first Release v0.1.0-rc
**Problem description**

With the already approved proposal and the included test plan we can proceed with the release process to have a version prior to Spring25 meta with version v0.1.0-rc

**Expected action**

Create PR to include changelog, readme and version.

### LINKED ISSUE #19: Restore Readme.md file
**Problem description**
Readme.md is deleted

**Expected behavior**
Restore Readme.md

**Alternative solution**
None

**Additional context**
@fernandopradocabrillo : Please try until eow (6th of Dec)

### PR DESCRIPTION
**Pull request by fernandopradocabrillo**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:
Include neccesary documentation to create the first public pre-release r1.1

#### Which issue(s) this PR fixes:

Fixes #17 
Fixes #19

### CODE REVIEW

**Review by FabrizioMoggio — DISMISSED** [review]

**Comment by hdamker** [comment]
@FabrizioMoggio Thanks, I will create the review issue within Release Management. 

And: I know it is confusing, but it is either a pre-release (if the the API version is an alpha or rc version) or a release of a public API version ... there is no such thing as a public pre-release. I edited the title of the PR accordingly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Include initial proposal for Most Frequent Location API - Telefónica by @fernandopradocabrillo in https://github.com/camaraproject/LocationInsights/pull/6
+* align-device-with-commonalities by @fernandopradocabrillo in https://github.com/camaraproject/LocationInsights/pull/9
+* Corrections in yaml examples for request body and response errors by @fernandopradocabrillo in https://github.com/camaraproject/LocationInsights/pull/11
+* Assumptions to use the API with postalCode but without countryName by @javier-carrocalabor in https://github.com/camaraproject/LocationInsights/pull/15
+* Include linter workflow files by @fernandopradocabrillo in https://github.com/camaraproject/LocationInsights/pull/16
+* Include test plan by @fernandopradocabrillo in https://github.com/camaraproject/LocationInsights/pull/13
+
+## New Contributors
+* @hdamker made their first contribution in https://github.com/camaraproject/LocationInsights/pull/1
+* @wrathwolf made their first contribution in https://github.com/camaraproject/LocationInsights/pull/2
```
Comment (hdamker) [inline CHANGELOG.md:41]: These two lines should be removed (the PRs are for good reason not listed above).
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Identity and Consent Management v0.2.1
+
+## most-frequent-location v0.1.0-rc.1
+
+**most-frequent-location v0.1.0-rc.1 is the 1st release candidate of the version 0.1.0**
+
+- 0.1.0-rc.1 Most Frequent Location API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/blob/r1.1/code/API_definitions/most-frequent-location.yaml)
```
Comment (hdamker) [inline CHANGELOG.md:26]: These links wouldn't work, correction (compare with README.md):

```suggestion
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/r1.1/code/API_definitions/most-frequent-location.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/LocationInsights/r1.1/code/API_definitions/most-frequent-location.yaml)
```

Hint: you can test the links by temporarily creating a branch `r1.1` (you would need to delete it before creating the release tag).

**Review by hdamker — CHANGES_REQUESTED** [review]
Looks generally good from Release Management point of view, but the links within CHANGELOG need to be corrected.

**Comment by hdamker** [comment]
A question about the version and the aim for Spring25: The [release tracker for 0.1.0](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/45056012/most-frequent-location+v0.1.0) is stating that the version 0.1.0 is targeted for Spring25 meta-release. For that meta-release the API should be based on Commonalities 0.5.0 (e.g. some changes to error schema) and you will have the public release around March 2025.

There two options which come to my mind (there might be others):
* doing now soon a public release of 0.1.0 outside of meta-release cycle and doing the adaptions for Spring25 in a 0.2.0
* releasing now an alpha version of 0.1.0 and doing the adaptions to Commonalities 0.5.0 until the release candidate

@fernandopradocabrillo Would be great to share your plan towards the Spring25 meta-release.

**Comment by fernandopradocabrillo** [comment]
> A question about the version and the aim for Spring25: The [release tracker for 0.1.0](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/45056012/most-frequent-location+v0.1.0) is stating that the version 0.1.0 is targeted for Spring25 meta-release. For that meta-release the API should be based on Commonalities 0.5.0 (e.g. some changes to error schema) and you will have the public release around March 2025.
> 
> There two options which come to my mind (there might be others):
> 
> * doing now soon a public release of 0.1.0 outside of meta-release cycle and doing the adaptions for Spring25 in a 0.2.0
> * releasing now an alpha version of 0.1.0 and doing the adaptions to Commonalities 0.5.0 until the release candidate
> 
> @fernandopradocabrillo Would be great to share your plan towards the Spring25 meta-release.

Hi @hdamker, you are right, I have to update the release tracker to v0.2.0.

The idea is to publish a first version aligned with Fall24 and the update the spec and align with Spring25 for a v0.2.0. At least in TEF we need an oficial version align with current agreements before going for the new one. I though I could have it before, but the WG is a bit short-staffed so it takes time 😄 

Thanks for pointing it out, I'll update the release tracker and proceed with the updates once I have the release created here. If possible, before the M3 due date.

**Comment by hdamker** [comment]
> I'll update the release tracker and proceed with the updates once I have the release created here. If possible, before the M3 due date.

Please keep the release tracker for 0.1.0, we are tracking them also [here](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/36012257/Sandbox+APIs+APIs+not+yet+in+any+meta-release). I've updated the release tracker accordingly.  For 0.2.0 please create an additional release tracker.

**Review by hdamker — APPROVED** [review]
LGTM - you are good to go as soon as there is a approval for the latest change from a @camaraproject/location-insights_codeowners

**Review by FabrizioMoggio — APPROVED** [review]
LGTM :-)

**Pull request merged**